automated/android/noninteractive-tradefed/setup.sh

```diff
@@ -13,14 +13,12 @@ if [ -n "${ANDROID_VERSION}" ] && echo "${ANDROID_VERSION}" | grep -q  "aosp-and
     java_path="/usr/lib/jvm/java-17-openjdk-amd64/bin/java"
 fi
 
-PKG_DEPS="coreutils usbutils curl wget zip xz-utils python-lxml python-setuptools python-pexpect aapt lib32z1-dev libc6-dev-i386 lib32gcc1 libc6:i386 libstdc++6:i386 libgcc1:i386 zlib1g:i386 libncurses5:i386 python-dev python-protobuf protobuf-compiler python-virtualenv python-pip python-pexpect psmisc"
-
```

Comment:
why removing all the PKG_DEPS?

Reply:
As you can see from @bhcopeland 's sugggestion, the packages are all installed within the docker image, no need to install them here again.

Reply:
right, but that make sense for you. but if I want to run this and I don't have it preinstalled I still need the PKG_DEPS.
So since you force people to use ubuntu, just check that its at least ubuntu:20.04... 
1. Then just check that its ubuntu:20.04.
2. default in your job submission SKIP_INSTALL: 'true'
3. if possible add the java-bits into the docker container ?
4. but that still needs to be here if its not installed...

Reply:
1. it's better not restricted to 20.04, without the check, it would be possible to be run within docker image like based Ubuntu 22.04
2. dropped SKIP_INSTALL, as it's not used anymore
3. JDK will be installed in the docker images side.
4. it will make the script here complicated to support more host versions, so drop the all the packages installation here, and let the user to prepare the host environments.